server/src/main/java/org/elasticsearch/action/admin/cluster/repositories/verify/VerifyRepositoryResponse.java

```diff
@@ -124,47 +106,29 @@ public int hashCode() {
         }
     }
 
-    private List<DiscoveryNode> nodes;
-
-    private ClusterName clusterName;
-
+    private List<NodeView> nodes;
 
     VerifyRepositoryResponse() {
     }
 
-    public VerifyRepositoryResponse(ClusterName clusterName, DiscoveryNode[] nodes) {
-        this.clusterName = clusterName;
-        this.nodes = Arrays.asList(nodes);
+    public VerifyRepositoryResponse(DiscoveryNode[] nodes) {
+        this.nodes = Arrays.stream(nodes).map(dn ->  new NodeView(dn.getId(), dn.getName())).collect(Collectors.toList());
     }
 
     @Override
     public void readFrom(StreamInput in) throws IOException {
         super.readFrom(in);
-        if (in.getVersion().onOrAfter(Version.V_6_4_0)) {
-            this.nodes = in.readList(NodeView::new).stream().map(n -> n.convertToDiscoveryNode()).collect(Collectors.toList());
-        } else {
-            clusterName = new ClusterName(in);
-            this.nodes = in.readList(DiscoveryNode::new);
-        }
+        this.nodes = in.readList(NodeView::new);
     }
 
     @Override
     public void writeTo(StreamOutput out) throws IOException {
         super.writeTo(out);
-        if (out.getVersion().onOrAfter(Version.V_6_4_0)) {
-            out.writeList(getNodes());
-        } else {
-            clusterName.writeTo(out);
-            out.writeList(nodes);
-        }
-    }
-
-    public List<NodeView> getNodes() {
-        return nodes.stream().map(dn -> new NodeView(dn.getId(), dn.getName())).collect(Collectors.toList());
+        out.writeList(nodes());
     }
 
-    public ClusterName getClusterName() {
-        return clusterName;
+    public List<NodeView> nodes() {
```

Comment:
I think you should keep the method named the same in 6.x and master. Also we *said* we'd prefer to go with getters and setters rather than the bare names for these things. As much as I personally like the names better, we said getters and setters.

Reply:
heh, i def forgot to mod this in 6.x, and i can make that its own PR. But if we prefer get/set then, im +1 to that as well.. I didnt know we had a strong pref toward get/set, I actually thought it was the opposite, even tho we had a lot more get/set in the codebase. Regardless, sync'ing w/ whats in 6.x is super important imho.